xsd/netex_framework/netex_genericFramework/netex_accessibility/netex_acsb_limitations.xsd

```diff
@@ -136,7 +137,11 @@ Rail transport, Roads and Road transport
 			<xsd:documentation>Whether a PLACE has step free access.</xsd:documentation>
 		</xsd:annotation>
 	</xsd:element>
-	<xsd:element name="EscalatorFreeAccess" type="LimitationStatusEnumeration" default="unknown">
+	<xsd:element name="StairFreeAccess" type="LimitationStatusEnumeration" default="unknown">
+		<xsd:annotation>
+			<xsd:documentation>Whether a PLACE has stair free access.</xsd:documentation>
```

Comment:
I suggest to add the difference with StepFree in het annotation: "Whether a PLACE has stair free access, in comparison with step free access one single step in the route is allowed".
With this addition the proposal is OK for me.